tests/unit/drivers/vector/test_marqo_vector_store_driver.py

```diff
@@ -193,7 +193,7 @@ def test_load_entries(self, driver, mock_marqo):
 
         # Assert
         assert len(entries) == 1
-        mock_marqo.index().search.assert_called_once_with("", limit=10000, filter_string=None)
+        mock_marqo.index().search.assert_called_once_with("", limit=10000)
```

Comment:
Add a test that specifies `namespace` so that the `filter_string` case is still handled.